install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -50,7 +50,11 @@ spec:
 {{- end }}
       containers:
         - name: istio-proxy
+{{- if contains "/" $.Values.global.proxy.image }}
```

Comment:
is the intent here if the image is a fully qualified repo/image/tag, to just use that directly?

Reply:
seems so